src/components/messageCanvas/messageCanvas.tsx

```diff
@@ -37,11 +37,7 @@ export default function MessageCanvas(props: MessageCanvasProps) {
         {props.children}
       </Card>
       <Box className="rustic-message-footer">
-        {props.getActionsComponent && (
-          <Card variant="outlined">
-            {props.getActionsComponent(props.message)}
-          </Card>
-        )}
+        {props.getActionsComponent && props.getActionsComponent(props.message)}
```

Comment:
This should not be changed. The component is expected to be several supported actions. Styling should be set here for the container.

Reply:
If we want to have the border in the `messageCanvas`, we need to hide it when no button is returned by `props.getActionsComponent`. Is it ok that I change the type of `getActionsComponent` from `(message: ThreadableMessage) => ReactNode` to `(message: ThreadableMessage) => ReactNode | undefined`?